cli/pcluster/config/param_types.py

```diff
@@ -434,6 +434,26 @@ def get_cfn_value(self):
 # that require some custom action during CFN/file conversion
 
 
+class ExtraJsonParam(JsonParam):
+    """Class to manage extra_json configuration parameters."""
+
+    def to_cfn(self):
+        """Convert param to CFN representation, if "cfn_param_mapping" attribute is present in the Param definition."""
+        if "cluster" in self.value:
+            self.value = {"cfncluster": self.value.pop("cluster")}
+        return super(ExtraJsonParam, self).to_cfn()
+
+    def to_file(self, config_parser):
+        """Set parameter in the config_parser in the right section.
+
+        The extra_json configuration parameter can contain both "cfncluster" or "cluster" keys but
+        we are writing "cluster" in the file to suggest the u
```

Comment:
You are missing the latest part of the comment: `... suggest the users the recommended syntax.`

Reply:
updated